packages/lib-classifier/src/store/ProjectStore.js

```diff
@@ -8,5 +8,12 @@ const ProjectStore = types
     resources: types.map(Project),
     type: types.optional(types.string, 'projects')
   })
+  .views(self => ({
+    get isVolumetricViewer() {
+      return (isValidReference(() => self.active))
+        ? self.active.experimental_tools.includes('volumetricViewer')
+        : false
+    }
+  }))
```

Comment:
This works and I can see the VolumetricViewer component when I run this branch locally. In our Tues meeting I recommended this view be on the Project store model, which can be confusing because there's Project.js and ProjectStore.js. I mentioned there are already two views on the Project store model, and those are in Project.js.

At this point I think it's half dozen of one and six of the other, but just wanted to be clear about what I recommended Tuesday. In ProjectStore.js, you have to use `isValidReference()` to grab the active `project`, whereas if the view is in Project.js, you just grab the active project via the `storeMapper()`. An example of the latter is `get display_name()` below (just an FYI, feel free to keep the view in ProjectStore.js if you prefer):

https://github.com/zooniverse/front-end-monorepo/blob/fbab1d8ef4e43b1f8fe0f43a27bea1c34e49d109/packages/lib-classifier/src/store/Project/Project.js#L24-L26

The presentational component SlideTutorialContainer uses `project.display_name` view:
https://github.com/zooniverse/front-end-monorepo/blob/fbab1d8ef4e43b1f8fe0f43a27bea1c34e49d109/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorialContainer.js#L12-L14

Reply:
Got this moved over to the proper file in latest commit - thanks for the clarification!